webapp/src/components/cardDetail/cardDetailContents.test.tsx

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import {fireEvent, render} from '@testing-library/react'
+
+import {IntlProvider} from 'react-intl'
+import {act} from 'react-dom/test-utils'
+
+import {DndProvider} from 'react-dnd'
+import {HTML5Backend} from 'react-dnd-html5-backend'
+
+import {TestBlockFactory} from '../../test/testBlockFactory'
+
+import CardDetailContents from './cardDetailContents'
+
+const wrapProviders = (children: any) => {
+    return (
+        <DndProvider backend={HTML5Backend}>
+            <IntlProvider locale='en'>{children}</IntlProvider>
+        </DndProvider>
+    )
+}
+
+global.fetch = jest.fn()
```

Comment:
The reason I added it is because Jest complains about not having it.
![image](https://user-images.githubusercontent.com/17804942/128346890-ff6413f4-80c6-4fd5-b669-ca1619ac8c9b.png)